quic/transport/packets/varints.nim

```diff
@@ -2,7 +2,7 @@ import std/math
 import pkg/stew/endians2
 
 type
-  VarIntCompatible* = range[0..2^62-1]
+  VarIntCompatible* = range[0'i64..2'i64^62-1]
```

Comment:
we generally avoid ranges: https://status-im.github.io/nim-style-guide/language.range.html

Reply:
It could be addressed in a new PR. What would you recommend in this case?